docs/dev_guide/index.rst

```diff
@@ -16,26 +16,54 @@ If you are thinking of contributing to sunpy please read the following carefully
 We currently recommend the :ref:`newcomers` as the place to start.
 This goes over the basics and has links to useful tutorials on git.
 
-.. toctree::
-   :maxdepth: 2
-
-   contents/newcomers
-   contents/code_standards
-   contents/documentation
-   contents/example_gallery
-   contents/tests
-   contents/pr_review_procedure
-   contents/api
-   contents/dependencies
-   contents/units_quantities
-   contents/new_objects
-   contents/extending_fido
-   contents/maintainer_workflow
-   contents/logger
-   contents/remote_data
-   contents/config
-   contents/funding
-   contents/ci_jobs
+.. panels::
+
+    Getting started
+    ^^^^^^^^^^^^^^^
+    .. toctree::
+      :maxdepth: 4
```

Comment:
The indentation depth on the theme is pretty large, it's really noticeable in this nested list.

Reply:
Drop it by 1?